bug-bounty-list/bug-bounty-list.json

```diff
@@ -5459,6 +5477,16 @@
     "hall_of_fame": true,
     "safe_harbor": "partial"
   },
+  {
+    "program_name": "Punchh",
+    "policy_url": "",
```

Comment:
@nightwatchcyber It looks like this field should be the other URL, and then you should have this as `submission_url`: https://hackerone.com/79b77b5d-abe5-4210-9d1f-7b63465d1d4e/embedded_submissions/new

Reply:
Fixed